tasks/main.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: Create consul group
+  group:
+    name: consul
+    system: yes
+
+- name: Create consul user
+  user:
+    name: consul
+    group: consul
+    createhome: no
+    system: yes
+
+- name: Mount "{{ consul_data_dir }}"
+  mount:
+    name: "{{ consul_data_dir }}"
+    src: "{{ consul_openstack_data_device }}"
+    fstype: ext4
+    state: mounted
+  when: consul_server
+
+- name: Create consul directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: consul
+    group: consul
+    mode: 0750
+  with_items:
+    - "{{ consul_config_dir }}"
```

Comment:
Typically all files in `/etc` have root as owner.  Using consul as the group is fine, but the owner of `/etc/consul.d` should be root.